tests/by-util/test_shuf.rs

```diff
@@ -147,6 +147,81 @@ fn test_head_count() {
     );
 }
 
+#[test]
+fn test_zero_head_count_pipe() {
+    let result = new_ucmd!()
+        .arg("-n0")
+        .pipe_in(vec![])
+        .succeeds();
+    result.no_stderr();
+
+    let result_seq: &str = result.stdout_str();
+    assert!(result_seq.is_empty(), "Output must be completely empty, not even newline");
```

Comment:
You can simplify it to:
```suggestion
    result.no_output();
```